test/small-integration-test.sh

```diff
@@ -1,10 +1,16 @@
-#!/bin/bash -e
+#!/bin/bash -ex
```

Comment:
Intentional change?  If so can you use `set -o xtrace` instead (and `set -o errexit`).